test/integration/dfmsv2_flow_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2021 ProximaX Limited. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+package integration
+
+import (
+	"crypto/rand"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/proximax-storage/go-xpx-chain-sdk/sdk"
+)
+
+func TestDriveV2FlowTransaction(t *testing.T) {
+	config, err := client.Network.GetNetworkConfig(ctx)
+	assert.Nil(t, err)
+
+	configDelta := 2
+	result := sendTransaction(t, func() (sdk.Transaction, error) {
+		return client.NewNetworkConfigTransaction(
+			sdk.NewDeadline(time.Hour),
+			sdk.Duration(configDelta),
+			config.NetworkConfig,
+			config.SupportedEntityVersions)
+	}, nemesisAccount)
+	assert.Nil(t, result.error)
+
+	waitForBlocksCount(t, configDelta)
+
+	const replicatorCount uint16 = 2
+	var replicators [replicatorCount]*sdk.Account
+	var storageSize uint64 = 500
+	var streamingSize uint64 = 100
+	var verificationFee = 100
+
+	for i := replicatorCount; i != 0; {
+		i--
```

Comment:
The decrement is placed in the loop because the index for the last element is _arraylength-1_.
```
const replicatorCount uint16 = 2
var replicators [replicatorCount]*sdk.Account

for i := replicatorCount; i != 0; {
    i--
    replicatorAccount, err := client.NewAccount()
    assert.Nil(t, err)
    fmt.Printf("replicatorAccount%d: %s\n", i, replicatorAccount)
    replicators[i] = replicatorAccount
}
```

Reply:
I've changed it to take the length of replicators instead of directly using the replicatorCount at https://github.com/proximax-storage/go-xpx-chain-sdk/pull/105/commits/a0b74d8e3cba5f9c5e9c0d7e45d15e1a4613d178

```
for i := 0; i < len(replicators); i++ {
	replicatorAccount, err := client.NewAccount()
	assert.Nil(t, err)
	fmt.Printf("replicatorAccount%d: %s\n", i, replicatorAccount)
	replicators[i] = replicatorAccount
}
```